deno-vite-plugin/tsconfig.json

```diff
@@ -4,8 +4,13 @@
     "declaration": true,
     "target": "ES2020",
     "module": "NodeNext",
-    "moduleResolution": "NodeNext"
+    "moduleResolution": "NodeNext",
+    "strict": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "isolatedModules": true,
+    "resolveJsonModule": true
   },
   "include": ["src"],
-  "exclude": ["vendor/"]
+  "exclude": ["vendor/", "node_modules"]
```

Comment:
without doing this second commit: https://github.com/commontoolsinc/labs/commit/ad0fc63429b344fcc693a2d4596a353319dedd72

there were typescript issues.. in deno-vite-plugin - because I added "@ai-sdk/xai"

@jsantell any ideas?

Reply:
humm, where did the vite updates come from?

Reply:
when I did [`deno add npm:@aisdk/xai` in toolshed](https://github.com/commontoolsinc/labs/pull/1076/files#diff-68b956ba9e8e1149408f2a1a5af3d14adca7ceca05605606de549b7b4454d447R14) walked through a lot of packages and made all the updates... perhaps I'm not using `add` in the proper way?